src/main/java/com/minecolonies/coremod/colony/jobs/JobBaker.java

```diff
@@ -0,0 +1,100 @@
+package com.minecolonies.coremod.colony.jobs;
+
+import com.minecolonies.coremod.client.render.RenderBipedCitizen;
+import com.minecolonies.coremod.colony.CitizenData;
+import com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton;
+import com.minecolonies.coremod.entity.ai.citizen.baker.EntityAIWorkBaker;
+import net.minecraft.nbt.NBTTagCompound;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.nbt.NBTTagCompound'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)